packages/api/test/core/index.test.ts

```diff
@@ -48,26 +52,32 @@ describe('APICore', function () {
       const operation = readme.spec.operation('/changelogs/{slug}', 'put');
 
       readme.setServer('https://dash.readme.com/api/v1');
-      expect(await readme.fetchOperation(operation, body, { slug })).to.deep.equal({
-        uri: '/api/v1/changelogs/new-release',
-        requestBody: body,
+      await readme.fetchOperation(operation, body, { slug }).then(({ data }) => {
+        expect(data).to.deep.equal({
+          uri: '/api/v1/changelogs/new-release',
+          requestBody: body,
```

Comment:
wait i'm also confused here lol, this is the response body?

Reply:
[(ignore me, see above)](https://github.com/readmeio/api/pull/539#discussion_r1006931204)